pym/bob/cmds/build.py

```diff
@@ -41,6 +42,24 @@ def hashWorkspace(step):
     return hashDirectory(step.getWorkspacePath(),
         os.path.join(step.getWorkspacePath(), "..", "cache.bin"))
 
+class LocalBuilderStatistic:
+    def __init__(self):
+        self.__activeOverrides = []
+        self.checkouts = 0
+        self.packagesBuilt = 0
+        self.packagesDownloaded = 0
+
+    def addOverrides(self, overrides):
+        for o in overrides:
+            if not o in self.__activeOverrides:
+                self.__activeOverrides.append(o)
+
+    def getActiveOverrides(self):
+        return self.__activeOverrides
+
+    def __getitem__(self, item):
+        return getattr(self, item)
```

Comment:
To access the members of the `LocalBuilderStatistic` class in `commonBuildDevelop`. Without I get:
```
  File "/home/biera/projects/bob/bob/pym/bob/cmds/build.py", line 967, in commonBuildDevelop
    + str(stats.packagesDownloaded) + " downloaded.")
TypeError: 'LocalBuilderStatistic' object is not subscriptable
```
Is there a better way to do this?